### ISSUE TITLE
Implement an example algorithm w/ swapper

### ISSUE THREAD

**Issue by tpendragon** [pulibrary/orangelight#3751]
Part of #3744 

This algorithm should be something fake and not realistic, so that any questions about the demo aren't about how the results are returned. Implementing a second algorithm is out of scope.

One example was "records that are also about cats are most important"

Blocked by implementation tickets.

Use the algorithm from #3816

**Comment by hackartisan** [comment]
I have an idea that "engineering" might be a better example boost than cats. There are a lot of results about a lot of different types of engineering, so I suspect it might demo more clearly. searching the catalog for engineering gives 500k results, whereas cats gives 40k results.

**Comment by tpendragon** [comment]
This will get closed by #3748 because we ended up implementing this feature via implementing an example, so we're closing this.
